Library/Formula/openal-soft.rb

```diff
@@ -5,13 +5,8 @@ class OpenalSoft < Formula
   revision 1
```

Comment:
You need to remove `revision 1`. And it would be great if you can remove redundant stable block to following form. Thanks.
```ruby
desc ".."
homepage ".."
url ".."
sha256 ".."
head ".."
```